kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/ShoppingTask.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.patterns.factory.tasks;
+
+public final class ShoppingTask implements Task {
+
+    private final String taskName;
+    private final String whatToBuy;
+    private final double quantity;
+    private boolean isTaskExecuted;
```

Comment:
a może klasa abstrakcyjna pomiędzy? AbstractTask, ponieważ pole isTaskExecuted jest w każdej z tych trzech klas. Ponadto nie nazywamy zmiennych od is... lepsza nazwa tutaj to poprostu `executed`.